test/functional/wallet_listsinceblock.py

```diff
@@ -423,6 +424,18 @@ def test_desc(self):
         coin_b = next(c for c in coins if c["amount"] == 2)
         assert_equal(coin_b["parent_descs"][0], multi_b)
 
+    def test_send_to_self(self):
+        """We can make listsinceblock output our change outputs."""
+        self.log.info("Test the inclusion of change outputs in the output.")
+
+        addr = self.nodes[2].getrawchangeaddress()
+        self.nodes[2].sendtoaddress(addr, 1)
+        self.generate(self.nodes[2], 1)
+        coins = self.nodes[2].listsinceblock()["transactions"]
+        assert_equal(len([c for c in coins if c["address"] == addr]), 0)
+        coins = self.nodes[2].listsinceblock(include_change=True)["transactions"]
```

Comment:
would this have two entries since you have the send(to change), and the normal change output as well?

I think the test should be explicit about it if so

Reply:
Yes. What would you suggest? Just asserting there are two entries?

Reply:
Extract both outputs, assert they are change addresses, I suspect is the best.

Reply:
Done. This needed a couple re-arrangements since `listsinceblock` lists also the "sent" coins, and we only want coins since the last block.

I couldn't resist a few drive-by cosmetic changes to the tests, including removing the unneeded block generation.